PyPDF2/_writer.py

```diff
@@ -794,6 +810,16 @@ def write(self, stream: StreamType) -> None:
         self._write_trailer(stream)
         stream.write(b_(f"\nstartxref\n{xref_location}\n%%EOF\n"))  # eof
 
+    def write(self, fileobj):
```

Comment:
```suggestion
    def write(self, fileobj: StrByteType) -> None:
```

We'll also want to add a docstring for this function as it's part of the public interface.